content/articles/getting-started-with-neural-network-in-pytorch/index.md

```diff
@@ -0,0 +1,287 @@
+- `nn.Sigmoid` - This is an activation function. It also takes in a matrix of inputs and outputs a matrix of outputs.
+- `nn.MSELoss` - This is a loss function. It takes in a matrix of outputs and targets and outputs a scalar.
+
+Let's begin by creating a neural network!
+
+#### Importing the necessary packages
+We need to import the `torch` library and other modules from the `torch` library:
+
+```python
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+import torch.utils.data as data
+import torchvision.datasets as datasets
+import torchvision.transforms as transforms
+```
+
+Let's go through them one by one to see what they do:
+
+```python
+import torch
+```
+This line imports the `torch` library. It is a high-level library that allows us to create neural networks. It is also a popular library for deep learning.
+
+```python
+import torch.nn as nn
+```
+
+Essentially, this imports the `nn` module from the `torch` library. This module contains all the building blocks for neural networks.
+
+```python
+import torch.nn.functional as F
+```
+
+This module has the `F` function used to create the activation functions.
+
+```python
+import torch.optim as optim
+```
+
+This line imports the `optim` module from the `torch` library. This module contains all the optimizers we can use to train our neural networks.
+
+```python
+import torch.utils.data as data
+```
+
+This module contains all the data loading and preprocessing functions that we can use to load and preprocess our data.
+
+```python
+import torchvision.datasets as datasets
+```
+
+This module contains all the datasets we can use to train our neural networks.
+
+```python
+import torchvision.transforms as transforms
+```
+
+This module contains all the transformations we can use to preprocess our data.
+
+#### Create a fully connected neural network
+Let's create a fully connected neural network. We will use the `nn.Linear` module to create a fully connected neural network:
+
+```python
+#imported modules here
+
+class Net(nn.Module):
+    def __init__(self, input_size, num_classes):#input_size will be 784, num_classes will be 10
+        super(Net, self).__init__()
+        self.fc1 = nn.Linear(input_size, 50)
+        self.fc2 = nn.Linear(50, num_classes)
+        
+    def forward(self, x):
+        x = F.relu(self.fc1(x))
+        x = self.fc2(x)
+        return x
+```
+Let's go through the code to see what it does.
+
+```python
+class Net(nn.Module):
+```
+
+In this line, we are creating a class called `Net`. This class is derived from the `nn.Module` class. This class is the base class for all the other modules.
+
+```python
+def __init__(self, input_size, num_classes):
+```
+The input size will be 784 since the amnesty image is 28x28 pixels. The `num_classes` will be ten since we want to classify the images into ten categories.
+
```

Comment:
I am wondering where the image came from. Every step should be linked to another for maximum value.